Packs/Core/Playbooks/playbook-Unsigned_and_unpopular_process_performed_an_injection.yml

```diff
@@ -0,0 +1,852 @@
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "12"
+      Weak:
+      - "8"
+    separatecontext: false
+    conditions:
+    - label: Weak
+      condition:
+      - - operator: notStartWith
+          left:
+            value:
+              simple: foundIncidents.name
+            iscontext: true
+          right:
+            value:
+              simple: Powershell Activity
+          ignorecase: true
+      - - operator: isNotEqualString
+          left:
+            value:
+              simple: foundIncidents.name
+            iscontext: true
+          right:
+            value:
+              simple: Unsigned process injecting into a Windows system binary with no command line
+          ignorecase: true
+      - - operator: notIn
+          left:
+            value:
+              complex:
+                root: foundIncidents.CustomFields
+                filters:
+                - - operator: isNotEqualString
+                    left:
+                      value:
+                        simple: foundIncidents.CustomFields.cid
+                      iscontext: true
+                    right:
+                      value:
+                        simple: alert.cid
+                      iscontext: true
+                accessor: action
+            iscontext: true
+          right:
+            value:
+              simple: Reported, BLOCKED
+          ignorecase: true
+      - - operator: notContainsGeneral
+          left:
+            value:
+              simple: foundIncidents.CustomFields.mitreattcktactic
+            iscontext: true
+          right:
+            value:
+              simple: TA0004 - Privilege Escalation
+          ignorecase: true
+      - - operator: notContainsGeneral
+          left:
+            value:
+              simple: foundIncidents.CustomFields.mitreattcktactic
+            iscontext: true
+          right:
+            value:
+              simple: TA0005 - Defense Evasion
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 300,
+          "y": 2030
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "12_13_Isolate": 0.42,
+      "12_8_#default#": 0.44,
+      "14_10_Yes": 0.37,
+      "17_10_yes": 0.22,
+      "17_5_#default#": 0.4,
+      "18_19_#error#": 0.65,
+      "21_12_#default#": 0.3,
+      "21_8_Weak": 0.32,
+      "4_12_Blocked": 0.32,
+      "4_15_Reported": 0.66,
+      "4_7_#default#": 0.81,
+      "5_14_#default#": 0.4,
+      "5_20_Signed": 0.12,
+      "7_10_yes": 0.2,
+      "7_17_#default#": 0.5
+    },
+    "paper": {
+      "dimensions": {
+        "height": 2785,
+        "width": 1660,
+        "x": -300,
+        "y": 30
+      }
+    }
+  }
```

Comment:
I do not see the limitation for marketplaceV2 only - this playbook should be released only to XSIAM

Reply:
Done